src/shapes/paint/trim_path.cpp

```diff
@@ -4,7 +4,8 @@
 
 using namespace rive;
 
-TrimPath::TrimPath() : m_TrimmedPath(makeRenderPath()) {}
+// TODO: how do we get a render-path here???
+TrimPath::TrimPath() : m_TrimmedPath(nullptr /*makeRenderPath()*/) {}
```

Comment:
Does this need to be resolved before we land?

Reply:
Definitely! --

Reply:
Fixed -- I now just pass-in the factory